.github/workflows/coverage.yml

```diff
@@ -1,7 +1,19 @@
 on:
   push:
     branches: [master]
+    paths-ignore:
```

Comment:
This is nice! What do you think of adding a workflow that runs `shellcheck` on all `*.sh`? It should already come pre-installed in GitHub actions I think, so all you should need is a new file that contains just an `actions/checkout@v2` step + 
```yaml
 - run:
     | shellcheck *.sh
```

Reply:
Thanks for the advice!

We shall definitely run shellcheck on workflow to ensure the scripts are written properly.

Reply:
Done!